client/wordpress-com.js

```diff
@@ -341,7 +341,14 @@ sections.push( {
 	module: 'login',
 	enableLoggedOut: true,
 	secondary: false,
-	isomorphic: true
+	isomorphic: {
+		cache: {
+			queryParams: {
+				include: [ 'client_id' ],
+				exclude: [ 'redirect_to' ],
+			}
```

Comment:
> `exclude` is a list of query parameters to ignore, they indicate that different values of this parameter shouldn't produce a different rendered page.

Conceptually, it seems a bit weird to essentially have a query arg and then say we aren't really visibly evaluating it, to the point that I'm believing it's such an edge case it doesn't deserve an option to control it.